std/algorithm/searching.d

```diff
@@ -3462,6 +3462,59 @@ unittest
 }
 
 /**
+Computes the first occurrence of `range`'s minimum (respectively maximum).
+
+Params:
+    pred = The ordering predicate to use to determine the extremum (minimum or
+           maximum) element.
+    range = The input range to search.
+
+Returns:
+    The position of the first encounter of the minimum (respectively maximum) in `range`.
+ */
+ptrdiff_t minPosIndex(alias pred = "a < b", Range)(Range range)
+    if (isInputRange!Range && !isInfinite!Range &&
+        is(typeof(binaryFun!pred(range.front, range.front))))
+{
+    if (range.empty) return -1;
+
+    int min_pos = 0, cur_pos = 0;
+    auto min = range.front();
+    for (range.popFront(); !range.empty; range.popFront())
+    {
+        ++cur_pos;
+        auto front = range.front();
```

Comment:
front should not have () parentheses - ditto for `min` initialisation.
It might be better to avoid copying `range.front` here.